store/evm_aux/store.go

```diff
@@ -39,7 +45,20 @@ func LoadStore() (*EvmAuxStore, error) {
 	if err != nil {
 		return nil, err
 	}
-	return NewEvmAuxStore(evmAuxDB), nil
+	dupEVMTxHashes := make(map[string]bool)
+	iter := evmAuxDB.NewIterator(
```

Comment:
Need a test to show this works as expected.

Reply:
Ok test added